Clockwise/ICircuitBreakerStorage.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Clockwise
+{
+    public interface ICircuitBreakerStorage : IDisposable
+    {
+        event EventHandler<CircuitBreakerStateDescriptor> CircuitBreakerStateChanged;
+        CircuitBreakerStateDescriptor StateDescriptor { get; }
```

Comment:
Holding a reference to the state feels like an implementation detail that we probably don't need and won't be able to honor in most scenarios.

Reply:
removed and replaced with method GetState